test/test_003_path_like_division.py

```diff
@@ -2,35 +2,35 @@
 
 
 def test_issue_003_path_like_division_trailing():
-	base = URI("http://example.com/foo/")
-	assert str(base) == "http://example.com/foo/"
-	assert str(base / "bar.html") == "http://example.com/foo/bar.html"
-	
-	base = URI("http://example.com/foo")
-	assert str(base) == "http://example.com/foo"
-	assert str(base / "bar.html") == "http://example.com/bar.html"
+    base = URI("http://example.com/foo/")
+    assert str(base) == "http://example.com/foo/"
+    assert str(base / "bar.html") == "http://example.com/foo/bar.html"
+
+    base = URI("http://example.com/foo")
+    assert str(base) == "http://example.com/foo"
+    assert str(base / "bar.html") == "http://example.com/bar.html"
 
 
 def test_issue_003_path_like_division_operators():
-	base = URI("http://example.com/foo/bar.html")
-	assert str(base / "baz.html") == 'http://example.com/foo/baz.html'
-	assert str(base // "cdn.example.com" / "baz.html") == 'http://cdn.example.com/baz.html'
-	assert str(base / "/diz") == 'http://example.com/diz'
-	assert str(base / "#diz") == 'http://example.com/foo/bar.html#diz'
-	assert str(base / "https://example.com") == 'https://example.com/'
+    base = URI("http://example.com/foo/bar.html")
+    assert str(base / "baz.html") == "http://example.com/foo/baz.html"
+    assert str(base // "cdn.example.com" / "baz.html") == "http://cdn.example.com/baz.html"
+    assert str(base / "/diz") == "http://example.com/diz"
+    assert str(base / "#diz") == "http://example.com/foo/bar.html#diz"
+    assert str(base / "https://example.com") == "https://example.com/"
```

Comment:
In a similar vein, I use double quotes for human text, single quotes for machine text. No qualms about making this hunk consistent, though! (Input v. output and I'd be getting silly with this pedantry. 😜)